lib/tasks/scrape.rake

```diff
@@ -64,6 +64,16 @@ def capitalize_name(name)
   end.join(' ')
 end
 
+def correct_typos(text)
+  text.split.map do |word|
+    typos_list[word.downcase] || word
+  end.join(' ')
+end
+
+def typos_list
+  @typos_list ||= YAML.load_file(Rails.root.join("lib/typos_list.yml"))
```

Comment:
i think this is ok for now, but i'm a little worried about creating all these methods and instance variables in the top level scope.
wdyt about creating a ticket to refactor this code and move it to a module/class?

Reply:
Yeah, I'm definitely up to it 👍